app/src/main/java/com/example/msogcardwar/Menu_Activity.java

```diff
@@ -3,18 +3,22 @@
 import androidx.appcompat.app.AppCompatActivity;
 
 import android.content.Intent;
+import android.media.MediaPlayer;
 import android.os.Bundle;
 import android.util.Log;
 import android.widget.Button;
 
 public class Menu_Activity extends AppCompatActivity {
 
     private Button start_game_btn;
+    private MediaPlayer backgroundSong;
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.menu_activity);
+        backgroundSong = MediaPlayer.create(Menu_Activity.this, R.raw.loyalty_freak_music04hello_regan);
```

Comment:
Again move it to init view ( we want to leave the onCreate the cleanest) personally I checked it, and he is right, the best practice is to leave this method the cleanest.